adafruit_hcsr04.py

```diff
@@ -119,51 +139,44 @@ def dist_cm(self):
         :return: Distance in centimeters.
         :rtype: float
         """
-        # This method only exists to make it easier to document. See either
-        # _dist_one_wire or _dist_two_wire for the actual implementation. One
-        # of those two methods will be assigned to be used in place of this
-        # method on instantiation.
-        pass
+        if self._trig is not None:
+            return self._dist_two_wire()
```

Comment:
How does it throw the runtime exception? A missing return is `return None` implicitly.

Reply:
at this time we only support two_wire so ill just remove the if() and we'll add one-wire later if i can find a PING sensor